packages/@lwc/rollup-plugin/src/index.ts

```diff
@@ -330,6 +330,17 @@ export default function lwc(pluginOptions: RollupLwcOptions = {}): Plugin {
             const [namespace, name] =
                 specifier?.split('/') ?? path.dirname(filename).split(path.sep).slice(-2);
 
+            /* v8 ignore next */
+            if (!namespace || !name) {
+                // TODO [#4824]: Make this an error rather than a warning
+                this.warn(
+                    'The component namespace and name could not be determined from the specifier ' +
+                        JSON.stringify(specifier) +
+                        ' or filename ' +
+                        JSON.stringify(filename)
+                );
+            }
```

Comment:
I can't figure out how to test this since we've only seen Vite do this.

Reply:
I'm using this same logic here:

https://github.com/cardoso/vite-plugin-lwc/blob/bf50de4bc4870b212efdae029c7611a5bc29f6a5/packages/cem-plugin-lwc/src/module.ts#L1-L6

And the unit test has been reported to fail on windows:

https://github.com/cardoso/vite-plugin-lwc/blob/bf50de4bc4870b212efdae029c7611a5bc29f6a5/packages/cem-plugin-lwc/__tests__/tagName.spec.ts#L1-L6

Could be related

Reply:
`path.sep` is different on both systems so it's quite possible we should just do `/` instead. I'm looking at `parseDescriptorFromFilePath` and it seems we only ever pass Rollup `identifier`s into it, and I'm wondering if Rollup just normalizes these to use `/` rather than `\` on Windows.

Reply:
The [Rollup docs](https://rollupjs.org/plugin-development/#transform) don't specify but given that this is an `id` I'm assuming it's the identifier of the imported modules, e.g. `import 'x/foo'` would be `"x/foo"`. So it's always going to be a forward slash. This is probably the issue.